libbpf-rs/src/netfilter.rs

```diff
@@ -0,0 +1,68 @@
+use std::mem::size_of;
+
+/// Netfilter protocol family for IPv4.
+pub const NFPROTO_IPV4: u32 = 2;
+/// Netfilter protocol family for IPv6.
+pub const NFPROTO_IPV6: u32 = 10;
+
+/// Netfilter hook number for pre-routing (0).
+pub const NF_INET_PRE_ROUTING: u32 = 0;
+/// Netfilter hook number for local input (1).
+pub const NF_INET_LOCAL_IN: u32 = 1;
+/// Netfilter hook number for packet forwarding (2).
+pub const NF_INET_FORWARD: u32 = 2;
+/// Netfilter hook number for local output (3).
+pub const NF_INET_LOCAL_OUT: u32 = 3;
+/// Netfilter hook number for post-routing (4).
+pub const NF_INET_POST_ROUTING: u32 = 4;
+
+#[derive(Clone, Debug, Default)]
+/// Options to be provided when attaching a program to a netfilter hook.
```

Comment:
Let's reorder comments before derives for consistency with the rest of the code, please.